src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -624,7 +624,7 @@
   <Target Name="CDefineChecker" BeforeTargets="Build" Condition="'$(CheckCDefines)'=='true'">
     <!-- Compiler Definition Verification -->
     <PropertyGroup>
-      <CMakeDefinitionSaveFile>$(IntermediateOutputPath)\cmake.definitions</CMakeDefinitionSaveFile>
+      <CMakeDefinitionSaveFile>$(IntermediateOutputPath)..\cmake.definitions</CMakeDefinitionSaveFile>
```

Comment:
Is this change related to EventPipe?

Reply:
I don't see why it would but, but my build and the CI builds were failing without it. I'll try to figure out what changed to start requiring this

Reply:
I figured this out. It actually fixes an indefinitely standing issue where the `check-definitions.py` script was always failing silently because it would catch the file not found error. I found it because I accidentally eliminated that catch (and since re added it). Unless it is objectionable to include this fix in this (unrelated) PR, I will leave it in